server/migrations/1612489825996_create-callback-field.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable max-len */
+/* eslint-disable quotes */
+const { dbClient } = require('../db/db.js');
+const { collections } = require('../db/schema.js');
+
+// Script used to extract the IDs which has the 7th column as 'Yes'
+
+// const { readFileSync } = require('fs');
+// const { join } = require('path');
+// const readXlsxFile = require('node-xlsx').default;
+
+// const file = readFileSync(join(__dirname,
+//   './assets/Batch 2 PEOIs Nov15-Jan3 - Callback Data MoH2 FINAL.xlsx'));
+// const xlsx = readXlsxFile.parse(file, { raw: true });
+
+// const ids = [];
+
+// xlsx[0].data.slice(1, xlsx[0].data.length - 1).forEach((row) => {
+//   if (row[6] === 'Yes') {
+//     ids.push(row[0]);
+//   }
+// });
+
+// console.log(ids.join(', '));
```

Comment:
Can this code be removed?

Reply:
Could be but I decided to let it in case we need to extract new Ids from a new spreadsheet again

Reply:
Just a nitpick: I'm inclined to get rid of this. Would be easy enough to filter in Excel or something and extract the IDs.